content/realtime/channels.textile

```diff
@@ -236,6 +236,147 @@ channel.subscribe { message in
 }
 ```
 
+h4(#rewind). Subscribing to a channel and fetching history with rewind
+
+If you wish to both subscribe to a channel, as well as obtain some messages from said channel's "history":/realtime/history seamlessly, you can make use of the "rewind":/realtime/channel-params feature. This will result in the specified number of messages from "history":/realtime/history being returned as part of the subscription process.
+
+minimize. View details
+  "Rewind":/realtime/channel-params has two methods of obtaining messages from history, either by getting a specified number of messages from history, or messages from a set period of time into the past.
+
+  In Ably libraries older than @v1.2@, you can specify the "rewind":/realtime/channel-params parameter in the "metadata":/realtime/channel-metadata of the channel name, tacking it onto the end of any metadata otherwise specified. For example, if you have no metadata and wish to subscribe to channel @my_channel@ and fetch the most recent message from "history":/realtime/history, you would specify the channel as @[?rewind=1]my_channel@. If the channel had some metadata, @[some_metadata]my_channel@, you would apply "rewind":/realtime/channel-params with @[some_metadata?rewind=1]my_channel@. For more details, look at our "channel paramater":/realtime/channel-params documentation.
```

Comment:
`tacking it onto the end` -> `appending it at the end` feels more general tech jargon IMO.